tests/providers/alibaba/cloud/hooks/test_oss.py

```diff
@@ -35,6 +35,7 @@ def setUp(self):
         try:
             self.hook = OSSHook(region=TEST_REGION, oss_conn_id=TEST_CONN_ID)
             self.hook.object_exists(key='test-obj', bucket_name=TEST_BUCKET)
+            print(self.hook.read_key(TEST_BUCKET, 'test-obj'))
```

Comment:
Can you add a real assertion here instead printing text?

Reply:
Sure, will fix it